cli/npm/common.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+/// Gets the corresponding @types package for the provided package name.
+pub fn types_package_name(package_name: &str) -> String {
```

Comment:
This will be used by byonm resolution in the future, so moved this up here from within the managed module.